src/selectors/appSettings.js

```diff
@@ -45,8 +46,8 @@ export function useBiometricsSelector() {
 
 export const maxPinCodeLengthSelector: Selector<number> = createSelector(
   ({ appSettings }) => appSettings,
-  (): number => {
-    // if (appSettings?.hasSixDigitsPin) return 6; // Temporary disable this
```

Comment:
I wonder why this was disabled in the first place

Reply:
Me too, maybe some other reason to disabled this